deployment/ansible/group_vars/development

```diff
@@ -12,6 +12,10 @@ services_ip: "{{ lookup('env', 'RF_SERVICES_IP') | default('33.33.40.80', true)
 
 postgresql_host: "{{ services_ip }}"
 
+s3_bucket: "raster-foundry-{{ lookup('env', 'USER') | regex_replace('^(.*\\\\\\)?(.*)$', '\\\\2') }}"
 sqs_queue: "raster-foundry-{{ lookup('env', 'USER') | regex_replace('^(.*\\\\\\)?(.*)$', '\\\\2') }}"
 sqs_dead_letter_queue: "raster-foundry-{{ lookup('env', 'USER') | regex_replace('^(.*\\\\\\)?(.*)$', '\\\\2') }}-dead-letter"
-s3_bucket: "raster-foundry-{{ lookup('env', 'USER') | regex_replace('^(.*\\\\\\)?(.*)$', '\\\\2') }}"
+logs_bucket: "raster-foundry-{{ lookup('env', 'USER') | regex_replace('^(.*\\\\\\)?(.*)$', '\\\\2') }}-logs"
+artifacts_bucket: "raster-foundry-{{ lookup('env', 'USER') | regex_replace('^(.*\\\\\\)?(.*)$', '\\\\2') }}-artifacts"
```

Comment:
`artifacts_bucket` needs to change into a variable that references a path inside of the `raster-foundry-releases` bucket, namespaced by tiler version number:

```bash
$ aws --profile rf-stg s3 ls s3://raster-foundry-releases/0.1.0/
2015-10-23 13:10:10          0
2015-10-23 13:10:28        466 bootstrap.sh
2015-10-23 13:10:28      15713 chunk.py
2015-10-23 13:10:28  125567836 mosaic.jar
```

This reference should be the same across all environments and users.